__fixtures__/example-todo-main/web/src/graphql/graphql.ts

```diff
@@ -15,6 +15,7 @@ export type Scalars = {
   Int: { input: number; output: number; }
   Float: { input: number; output: number; }
   BigInt: { input: any; output: any; }
+  Byte: { input: any; output: any; }
```

Comment:
Is it Byte or Bytes?

```
   const prismaScalarToTsType = {
     String: 'string',
 @@ -308,6 +308,7 @@ export const mapPrismaScalarToPagePropTsType = (scalarType) => {
     Float: 'number',
     Decimal: 'number',
     DateTime: 'string',
     Bytes: 'Buffer',
```

Reply:
Rather, I should ask: why is it Byte singular and not Bytes?

Reply:
The GraphQL type is `Byte`
The Prisma type is `Bytes`
The TS type is `Buffer`

Reply:
Oh, in Prisma is Bytes and GraphQL scalar is Byte?

Reply:
I see now: ```
   const prismaTypeToGraphqlType = {
     Json: 'JSON',
     Decimal: 'Float',
     Bytes: 'Byte',```